.babelrc

```diff
@@ -1,6 +1,26 @@
 {
-  "presets": [
-    "env",
-    "react"
-  ]
+  "env": {
+    // Compatibility Profile. ES5 output and CommonJS module format.
+    "commonjs": {
+      "presets": [
+        ["env", {
+          "useBuiltIns": false
+        }],
+        "react"
+      ]
+    },
+    // Future Profile. ES6 output and module syntax.
+    "es": {
+      "presets": [
+        ["env", {
+          "useBuiltIns": false,
+          "modules": false,
+          "targets": {
+            "node": "6.5"
+          }
+        }],
+        "react"
+      ]
+    }
+  }
 }
```

Comment:
Care with no `end of line`. Missing add `.editorconfig` to this project?

Reply:
Yes, that is something I missed too. No linters and no `.editorconfig`. I supposed that given it is a tiny project you did not want to add extra complexity, but I will be happy to add it if you prefer so. Do you?

Reply:
Yes, you are right, it's a tiny project. In my opinion, `.editorconfig` has 0 configuration and it gives too much. Linters could be more difficult to implement because rules depends by type of project.

For me it's ok if we add `.editorconfig` as you wish

Reply:
Sure @Nasdan, I have include it. Thanks for the tip!